### DISCUSSION TITLE
Welcome to stdlib Discussions!

### DISCUSSION THREAD

**Discussion by kgryte** [stdlib-js/stdlib#370] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by akash8843** [comment]
Hello @kgryte sir , 

My name is Akash Shukla, and I’m an aspiring software engineer passionate about web development. I have hands-on experience with HTML, CSS, JavaScript, Bootstrap, and Node.js, and I’ve been actively learning through real-world projects. 

I’m really interested in contributing to the stdlib project for GSOC 2025. However, I feel like I might be a bit late in starting my contributions, and I’m worried about catching up. 

Could you please guide me on how I can still contribute and where I should begin? I’m eager to learn, work hard, and make meaningful contributions to the project with your support. 

Looking forward to your response! 

Best,  
Akash Shukla

**Reply by kgryte** [reply]
@akash8843 Please see https://github.com/stdlib-js/google-summer-of-code/tree/main.

**Comment by akash8843** [comment]
Hi @kgryte (Athan),

I hope you're doing well! I've created a pull request that updates comments in two files to improve clarity and documentation. All tests have passed, and code coverage remains at 100%.

Whenever you get a chance, could you please review the pull request? Additionally, the workflows in the PR require manual approval, so I would appreciate it if you could take a look at those too.

Here’s the link to the PR: -akash8843:fix-quantile-comments

Thank you so much for your time and guidance! Please let me know if there’s anything else you’d like me to address.

Best regards, Akash

**Comment by akash8843** [comment]
Subject: Request for Review: Pull Request for Issue #5901

Hello @kgryte (Athan),

I hope you are doing well!

I have submitted a pull request to resolve Issue #5901 (Refactor random number generation for benchmark consistency). I have refactored the random number generation logic to use the random/base/uniform module, ensuring better code clarity and alignment with the existing standards in the stdlib package.

I kindly request you to review my PR when you get a chance. Please let me know if there are any improvements or changes needed — I’ll be happy to make adjustments!

Thank you very much for your guidance and support.

Best regards,
Akash Shukla

**Comment by akash8843** [comment]
Subject: Request for Code Review — Fix for Issue #5895

Hello @kgryte ,

I have worked on resolving issue #5895 and have submitted a pull request for the changes. I have made the necessary fixes, updated the comments, and ensured that the code aligns with the project's guidelines.

Could you please review my pull request whenever you have time? Let me know if there are any improvements or adjustments needed.

Thank you for your guidance and support!

Best regards,
Akash Shukla
